chimeraos/airootfs/root/install.sh

```diff
@@ -28,10 +28,11 @@ fi
 #### Post install steps for system configuration
 # Copy over all network configuration from the live session to the system
 MOUNT_PATH=/tmp/frzr_root
-if [ -d "/etc/NetworkManager/system-connections" ]; then
-    mkdir -p -m=700 ${MOUNT_PATH}/etc/NetworkManager/system-connections
-    cp  /etc/NetworkManager/system-connections/* \
-        ${MOUNT_PATH}/etc/NetworkManager/system-connections/.
+SYS_CONN_DIR="/etc/NetworkManager/system-connections"
+if [ -d $SYS_CONN_DIR && "$(ls -A $SYS_CONN_DIR)" ]; then
```

Comment:
can we just copy the whole directory instead?

Reply:
I would not copy the directory since it is part of network manager package. I prefer to copy the contents instead since they are created by NM itself.

https://archlinux.org/packages/extra/x86_64/networkmanager/